### ISSUE TITLE
Bulk operations on Data Libraries

### ISSUE THREAD

**Issue by bebatut** [galaxyproject/galaxy#12209]
Hi, 

I was interacting with Data libraries yesterday and it was quite annoying to not be able to modify the permissions for all datasets in a folder in bulk. I had to do it manually one-by-one.

Would it be possible to have some bulk operations in data libraries?

ping @OlegZharkov

**Comment by OlegZharkov** [comment]
Yeah, that's a bigger project. It requires some backend love. We probably need to create `manage all` button. 
A future project @davelopez  :sweat_smile:
